src/pipeline.ml

```diff
@@ -509,47 +527,42 @@ module Ocaml_org = struct
     in
     pipelines, additional_build_args
 
-  let extras () =
-    let tarsnap =
-      let monthly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 30) () in
-      Current_ssh.run ~schedule:monthly "watch.ocaml.org" ~key:"tarsnap" (Current.return ["./tarsnap-backup.sh"])
-    in
-    let peertube =
-      let weekly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 7) () in
-      let image = Cluster.Watch_docker.pull ~schedule:weekly "chocobozzz/peertube:production-bookworm" in
-      Cluster.Watch_docker.service ~name:"infra_peertube" ~image ()
-    in
-    [tarsnap; peertube]
+  let watch_ocaml_org = "watch.ocaml.org"
+  module Watch_docker = Current_docker.Make(struct let docker_context = Some watch_ocaml_org end)
 
   let v ?app ?notify:channel ?filter ~sched ~staging_auth () =
     (* [web_ui collapse_value] is a URL back to the deployment service, for links
       in status messages. *)
     let web_ui repo = Uri.with_query' base_url ["repo", repo] in
-    let build (org, name, builds) =
-      Cluster_build.repo ?channel ~web_ui ~org ~name builds
-    in
-    let build_for_registry ?additional_build_args (org, name, builds) =
-      Build_registry.repo ?channel ?additional_build_args ~web_ui ~org ~name builds
```

Comment:
Rather than constructing a closure and then mapping it, we just construct the values we need in `docker_registry_pipelines`.